src/app/code/community/FireGento/Pdf/Model/Observer.php

```diff
@@ -46,6 +47,36 @@ public function addInvoiceNotes(Varien_Event_Observer $observer)
     }
 
     /**
+     * Add notes based on shipping country
+     *
+     * @param  Varien_Event_Observer $observer observer object
+     *
+     * @return $this
+     */
+    public function addShippingCountryNotes(Varien_Event_Observer $observer)
+    {
+        $order = $observer->getOrder();
+        $shippingCountryId = $order->getShippingAddress()->getCountryId();
+        $countryNotes = unserialize(Mage::getStoreConfig('sales_pdf/invoice/shipping_country_notes'));
```

Comment:
I think we should have here a model to encapsulate all this with one method `getNoteForCountry()`